### PR TITLE
Added installation steps for windows and linux

### PR DESCRIPTION
**Pull request by mohan08p**
Terraform Installation steps for windows and linux added.

We will need to change url for windows and linux in README.md of os directry respectively once this is verified and merge.

Thank you,
Mohan

### CODE REVIEW

File: beginners/os/linux/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Installing Terraform on Linux
+
+A binary distribution is avaialble for all environments. Let's grab the latest version of it for linux.
+
+```
+$ wge thttps://releases.hashicorp.com/terraform/0.12.26/terraform_0.12.26_linux_amd64.zip
```
Comment (RastogiAbhijeet) [inline beginners/os/linux/README.md:6]: It's supposed to be `wget` I feel.

File: beginners/os/linux/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+Then unzip the archieve,
+
+```
+$ unzip terraform_0.12.26_windows_amd64.zip
```
Comment (RastogiAbhijeet) [inline beginners/os/linux/README.md:12]: terraform_0.12.26_<strike>windows</strike>linux_amd64.zip

**Comment by collabnix** [comment]
Thanks for contributing.

File: beginners/os/linux/README.md (hunk 1 above)
Comment (mohan08p) [reply beginners/os/linux/README.md:6]: yeah.. thanks for correcting.

**Pull request merged**